openfl-workspace/keras_nlp/code/dataloader_utils.py

```diff
@@ -0,0 +1,213 @@
+"""Copyright (C) 2020 Intel Corporation.
+
+Licensed subject to the terms of the separately executed evaluation
+license agreement between Intel Corporation and you.
+"""
+import numpy as np
+
+from logging import getLogger
+
+from os import path, getcwd, remove
+from zipfile import ZipFile
+import subprocess
+
+logger = getLogger(__name__)
+
+
+def download_data_():
+    """Download data.
+
+    Returns:
+      string: relative path to data file
+    """
+    pkg = "fra-eng.zip"   # Language file: change this to change the language
+    data_dir = "data"
+    url = "http://www.manythings.org/anki/" + pkg
+    filename = pkg.split("-")[0] + ".txt"
+
+    workspace_dir = getcwd()
+    default_path = path.join(workspace_dir, data_dir)
+    pkgpath = path.join(default_path, pkg)       # path to downloaded zipfile
+    filepath = path.join(default_path, filename)  # path to extracted file
+
+    if path.isfile(filepath):
+        return path.join(data_dir, filename)
+    try:
+        subprocess.run(["wget", url, "-P", default_path], check=True)
```

Comment:
@walteriviera instead of calling `wget` directly through the shell, could you call `urlretrieve` from urllib as in [this example](https://github.com/intel/openfl/blob/develop/openfl-workspace/torch_cnn_histology/code/histology_utils.py#L39)?

Reply:
Also after downloading the file, it's integrity should be checked: https://github.com/intel/openfl/blob/develop/openfl-workspace/torch_cnn_histology/code/histology_utils.py#L40-L41

Reply:
Fixed. Thanks Patrick 👍